integreat_cms/cms/templates/dashboard/dashboard.html

```diff
@@ -10,7 +10,7 @@ <h1 class="heading">
         <div class="xl:flex gap-3 pt-2">
             <div class="xl:w-1/2">
                 <!-- Task Widget -->
-                {% if perms.cms.change_page or perms.cms.change_feedback %}
+                {% if (perms.cms.change_page or perms.cms.change_feedback) and request.region.default_language %}
```

Comment:
Parentheses are not allowed in Django template conditions, you need to stack two separate `{% if %}` tags if you need something like `if (A or B) and C`.
If the condition would be `if A or (B and C)`, you could just write `if A or B and C`, since `and` has higher precedence.

Reply:
Yes, I found that out just now